hack/debug.sh

```diff
@@ -253,24 +275,42 @@ do_pod_service_connectivity_check () {
     echo ""
 }
 
-do_node () {
-    config=$(ps wwaux | grep -v grep | sed -ne 's/.*openshift.*--config=\([^ ]*\.yaml\).*/\1/p')
+get_config_path_from_service() {
+    # 'node', 'master', 'master api', 'master controllers'
+    service_type=$1
+    # 'atomic-openshift-node.service', 'atomic-openshift-master-controllers.service', etc
+    service_name=$2
+    config=$(ps wwaux | grep -v grep | sed -ne "s/.*openshift start ${service_type} --config=\([^ ]*\.yaml\).*/\1/p")
```

Comment:
This will not match in some cases where the service uses other cli options before --config.
Example: /usr/bin/openshift start node --loglevel=5 --config=/...
ps wwaux | grep -v grep | sed -ne "s/.*openshift start node --.* --config=\([^ ]*\.yaml\).*/\1/p" ?